### PR TITLE
Add community command

### PR DESCRIPTION
**Pull request by paulasjes-stripe**
### Reviewers
r? @tomer-stripe 
cc @stripe/developer-products

 ### Summary

Adds a new `community` command which directs users to the Stripe Developer Community Discord server.

Screenshot 2021-07-01 at 3 39 15 PM

### CODE REVIEW

File: pkg/cmd/community.go [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+
+type communityCmd struct {
+	cmd *cobra.Command
+}
+
+func newCommunityCmd() *communityCmd {
+	cc := &communityCmd{}
+
+	cc.cmd = &cobra.Command{
+		Use:     "community",
```
Comment (tomer-stripe) [inline pkg/cmd/community.go:25]: Do we want to call this `community` or `chat`? You could add an alias using `Aliases: []string{"discord", ...}`

File: pkg/cmd/community.go [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+		Short:   "Chat with Stripe engineers and other developers",
+		Long:    "Chat with Stripe engineers and other developers in the official Stripe Developer Community Discord server",
```
Comment (tomer-stripe) [inline pkg/cmd/community.go:27]: We don't need a `Long` if it doesn't add anything materially different.

File: pkg/cmd/community.go [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+	}
+
+	fmt.Printf("Chat with other developers and Stripe engineers in the official Stripe Discord server.\n\nPress Enter to open the browser or visit %s", communityURL)
+
+	input := os.Stdin
+	fmt.Fscanln(input)
+
+	err := openBrowser(communityURL)
+	if err != nil {
+		fmt.Printf("Failed to open browser, please go to %s manually.", communityURL)
```
Comment (tomer-stripe) [inline pkg/cmd/community.go:48]: We should probably return an error here instead of printing it.

**Review by tomer-stripe — COMMENTED** [review]
Couple small nits, overall looks good!

File: pkg/cmd/community.go (hunk 1 above)
Comment (paulasjes-stripe) [reply pkg/cmd/community.go:25]: I felt that `community` was probably the most descriptive, but I've now added aliases for `discord` and `chat`.

File: pkg/cmd/community.go (hunk 3 above)
Comment (paulasjes-stripe) [reply pkg/cmd/community.go:48]: I was copying the behaviour here: https://github.com/stripe/stripe-cli/blob/6e96f25437fc9e3a26924a49e9e970c331c6ecea/pkg/login/client_login.go#L66-L71

Comment (paulasjes-stripe) [reply pkg/cmd/community.go:48]: Happy to return an error though instead if you feel that's more appropriate.

File: pkg/cmd/community.go (hunk 1 above)
Comment (tomer-stripe) [reply pkg/cmd/community.go:25]: Sounds good! Do the DA folks have feelings on this? Could imagine a future where `community` might want to go to the community experts or something.

File: pkg/cmd/community.go (hunk 3 above)
Comment (tomer-stripe) [reply pkg/cmd/community.go:48]: Ah alright. That's fine for now!

**Comment by tomer-stripe** [comment]
Looks good to me, really appreciate this Paul!

**Review by tomer-stripe — APPROVED** [review]

**Pull request merged**